content/docs/guides/bemi.md

```diff
@@ -0,0 +1,158 @@
+
+```sql
+GRANT USAGE ON SCHEMA public TO <role_name>;
+
+-- Grant SELECT access to tables for selective tracking
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO <role_name>;
+
+-- Grant SELECT access to new tables created in the future for selective tracking
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO <role_name>;
+```
+
+## Create a replication slot and publication
+
+Run these commands to create a Postgres publication and add the required replication identity:
+
+```sql
+-- Create "bemi" PUBLICATION to enable logical replication
+CREATE PUBLICATION bemi FOR ALL TABLES;
+
+-- Create a procedure to set REPLICA IDENTITY FULL for tables to track the "before" state on DB row changes
+CREATE OR REPLACE PROCEDURE _bemi_set_replica_identity() AS $$ DECLARE current_tablename TEXT;
+BEGIN
+  FOR current_tablename IN SELECT tablename FROM pg_tables WHERE schemaname = 'public' LOOP
+    EXECUTE format('ALTER TABLE %I REPLICA IDENTITY FULL', current_tablename);
+  END LOOP;
+END $$ LANGUAGE plpgsql;
+-- Call the created procedure
+CALL _bemi_set_replica_identity();
+```
+
+## Create a Postgres connection to Bemi
+
+1. Create a [new database connection](https://dashboard.bemi.io/databases/source/new) by entering the connection details for your Neon database. You can get these details from your Neon connection string, which you'll find in the **Connection Details** widget on the **Dashboard** of your Neon project. For example, given a connection string like this:
+
+    ```sql
+    postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname?sslmode=require
+    ```
+
+    Enter the details in the **Connect PostgreSQL Database** dialog as shown below. Your values will differ.
+
+    - **Host**: ep-cool-darkness-123456.us-east-2.aws.neon.tech
+    - **Port**: 5432
+    - **Database Name**: dbname
+    - **Username**: alex
+    - **Password**: AbC123dEf
+
+    ![Bemi Connect PostgreSQL Database](/docs/guides/bemi_connect_postgres.png)
+
+2. During the connection setup or any time after, you can configure the tables you want to track changes for:
+
+    ![Bemi Tracked Tables](/docs/guides/bemi_tracked_tables.png)
+
+3. Please wait a few minutes while Bemi provisions the infrastructure. Once this succeeds, you’ve successfully configured a Bemi Postgres source for your Neon database.
+
+## Allow inbound traffic
+
+If you’re using Neon's IP Allow feature to limit IP addresses that can connect to Neon, you will need to allow inbound traffic from Bemi. [Contact Bemi](mailto:hi@bemi.io) to get the static IPs that need to be allowlisted. For information about configuring allowed IPs in Neon, see [Configure IP Allow](/docs/manage/projects#configure-ip-allow).
+
+ ## **References**
```

Comment:
this is too bold when previewed
<img width="575" alt="Screenshot 2024-06-01 at 4 13 22 PM" src="https://github.com/neondatabase/website/assets/46300090/7e64979c-6b84-4320-a3e5-80e8d8443811">

Reply:
Thank you. Fixed.